src/naive/datetime/mod.rs

```diff
@@ -1289,6 +1345,37 @@ impl AddAssign<OldDuration> for NaiveDateTime {
     }
 }
 
+impl Add<Months> for NaiveDateTime {
+    type Output = NaiveDateTime;
+
+    /// An addition of months to `NaiveDateTime` clamped to valid days in resulting month.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the resulting date would be out of range.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use chrono::{Duration, NaiveDateTime, Months};
+    /// use std::str::FromStr;
+    ///
+    /// let dt = NaiveDateTime::from_str;
+    /// let m = Months::new;
+    ///
+    /// assert_eq!(dt("2014-01-01T01:00:00").unwrap() + m(1), dt("2014-02-01T01:00:00").unwrap());
+    /// assert_eq!(dt("2014-01-01T00:02:00").unwrap() + m(11), dt("2014-12-01T00:02:00").unwrap());
+    /// assert_eq!(dt("2014-01-01T00:00:03").unwrap() + m(12), dt("2015-01-01T00:00:03").unwrap());
+    /// assert_eq!(dt("2014-01-01T00:00:04").unwrap() + m(13), dt("2015-02-01T00:00:04").unwrap());
+    /// assert_eq!(dt("2014-01-31T00:05:00").unwrap() + m(1), dt("2014-02-28T00:05:00").unwrap());
+    /// assert_eq!(dt("2020-01-31T06:00:00").unwrap() + m(1), dt("2020-02-29T06:00:00").unwrap());
+    /// ```
+    fn add(self, rhs: Months) -> Self::Output {
+        let new_date = self.date() + rhs;
+        new_date.and_time(self.time())
```

Comment:
```rust
Self {
    date: self.date.checked_add_months(rhs).unwrap(),
    time: self.time,
}
```

Reply:
This doesn't seem to be resolved?

Reply:
Sorry, I fixed it in the other two places and forgot to fix it here. It is resolved now.